build-logic/src/main/kotlin/io/github/droidkaigi/confsched2023/primitive/VersionCatalog.kt

```diff
@@ -0,0 +1,28 @@
+package io.github.droidkaigi.confsched2023.primitive
+
+import org.gradle.api.Project
+import org.gradle.api.artifacts.ExternalModuleDependencyBundle
+import org.gradle.api.artifacts.MinimalExternalModuleDependency
+import org.gradle.api.artifacts.VersionCatalog
+import org.gradle.api.artifacts.VersionCatalogsExtension
+import org.gradle.kotlin.dsl.getByType
+import org.gradle.plugin.use.PluginDependency
+
+internal val Project.libs: VersionCatalog
+    get() = extensions.getByType<VersionCatalogsExtension>().named("libs")
+
+internal fun VersionCatalog.version(name: String): String {
+    return findVersion(name).get().requiredVersion
+}
+
+internal fun VersionCatalog.library(name: String): MinimalExternalModuleDependency {
```

Comment:
Now I see why we can do this 👀

Reply:
we can call version catalog more shortly and not depends on Java/Optional :yay: